management/commands/calculate_all_fees.py

```diff
@@ -42,6 +42,12 @@ def handle(self, *args, **options):
                     'country': old_band.country,
                     'currency': old_band.currency,
                 })
+                if new_band_form.is_valid():
+                    new_band = new_band_form.save(commit=bool(options['save']))
```

Comment:
I think elsewhere we use the `store_true` action with a default so the resulting option doesn't neet to be cast.

Reply:
Good call--I've made the change.